src/jukebox/jukebox/speaking_text/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""
+Text to Speech. Plugin to speak any given text via speaker
+"""
+
+import logging
+import jukebox.cfghandler
+import shlex
+import subprocess
+
+logger = logging.getLogger('jb.speaking_text')
+cfg = jukebox.cfghandler.get_handler('jukebox')
+
+
+def run_unix_command(command):
+    cmds = shlex.split(command)
+    return subprocess.Popen(cmds, start_new_session=True)
+
+
+# http://espeak.sourceforge.net/commands.html
+def with_espeak(text, params):
```

Comment:
> The content of params is unknown to a potential user.
> 
> Proposal:
> 
> ```
> def with_espeak(text, speed=None, speak_punctuation=None, ...)
> 
> ...
> 
> if speed is None:
>   speed_option = cfg.setndefault('speaking_text', speed', value='normal')
> ```

I see your point. But I didn't want to do it that way.. reason was, let's say you only want to update `speed` which is the last of the params, you would have to provide the others as well .. with `params` (or maybe `**kwargs`?) as a dictionary, this could be avoided.